sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/keyvault/common/AzureKeyVaultConfiguration.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.autoconfigure.keyvault.common;
```

Comment:
why not put it in `com.azure.spring.cloud.autoconfigure.keyvault`, to be consistent with storage?

Reply:
updated